src/Magnum/ImGuiIntegration/Widgets.h

```diff
@@ -0,0 +1,66 @@
+#ifndef Magnum_ImGuiIntegration_Widgets_h
+#define Magnum_ImGuiIntegration_Widgets_h
+/*
+    This file is part of Magnum.
+
+    Copyright © 2010, 2011, 2012, 2013, 2014, 2015, 2016, 2017, 2018
+              Vladimír Vondruš <mosra@centrum.cz>
+    Copyright © 2018 ShaddyAQN <ShaddyAQN@gmail.com>
+    Copyright © 2018 Tomáš Skřivan <skrivantomas@seznam.cz>
+    Copyright © 2018 Jonathan Hale <squareys@googlemail.com>
+
+    Permission is hereby granted, free of charge, to any person obtaining a
+    copy of this software and associated documentation files (the "Software"),
+    to deal in the Software without restriction, including without limitation
+    the rights to use, copy, modify, merge, publish, distribute, sublicense,
+    and/or sell copies of the Software, and to permit persons to whom the
+    Software is furnished to do so, subject to the following conditions:
+
+    The above copyright notice and this permission notice shall be included
+    in all copies or substantial portions of the Software.
+
+    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+    IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+    FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+    THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+    LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+    FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+    DEALINGS IN THE SOFTWARE.
+*/
+
+/** @file
+ * @brief Function @ref Magnum::ImGuiIntegration::MgImGui::Image()
+ */
+
+#include <Magnum/Math/Vector2.h>
+#include <Magnum/Math/Vector4.h>
+#include <Magnum/Math/Color.h>
+#include <Magnum/GL/Texture.h>
+
+#include "Magnum/ImGuiIntegration/visibility.h"
+#include "Magnum/ImGuiIntegration/Conversion.h"
+
+#include <imgui.h>
+
+namespace Magnum { namespace ImGuiIntegration { namespace MgImGui {
```

Comment:
Oh, haha, somehow forgot to comment on this namespace (wasn't there a discussion about the name before?). 

What about having `Image()` directly in `ImGuiIntegration`? (also why not `image()`, lowercase?). Another idea is have the namespace named `Widgets`, but since here is just a single function right now, I would place it directly inside the parent namespace.

Reply:
I commented on this only via gitter. My arguments were:

 - `ImGui::Image` would be best, as it would match `ImGui::Image`, but just overload the texture parameter
 - That doesn't work, though, as otherwise I would have to `::` prefix all ImGui calls, which looks pretty ugly
 - Hence I though to somewhat still match the style of ImGui calls and to make those easily distinguishable, `MgImGui::Image` would look nice and that same namespace could contain other future ImGui widgets

Reply:
> as otherwise I would have to :: prefix all ImGui calls

but only in the code of the integration itself and only for `Image` calls, no? That could be still acceptable I think. 

I would go with `ImGuiIntegration::image()`, since that's how we name functions here (similarly, we also don't use `snail_case` for APIs that extend / emulate STL, so why emulate ImGui naming). Or wrap it in `ImGuiIntegration::Widgets::image()`, but that's too much of a colon cancer for my taste, especially considering there's just a single function right now. `Magnum::Ui` also doesn't have any `Widgets` subnamespace either.